pkg/proto/configuration/grpc/grpc.proto

```diff
@@ -35,6 +35,14 @@ message ClientConfiguration {
   //
   // Header names must be lower case.
   repeated string forward_metadata = 4;
+
+  // Map of gRPC metadata headers to set in client connection
```

Comment:
Nit: General remark (also applies to #65). Please add all sentences in comments with a full stop (a period). Thanks!

Reply:
done